build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -530,7 +597,7 @@ def merge(self, other):
   def to_json_str(self) -> str:
     json_object = {"commit": self.commit, "benchmarks": []}
     json_object["benchmarks"] = [b.to_json_object() for b in self.benchmarks]
-    return json.dumps(json_object)
+    return json.dumps(json_object, indent=4)
```

Comment:
We usually use `indent=2` (e.g. https://github.com/openxla/iree/blob/50a6c175b1a5166742b4524a84c1b56931bc2993/build_tools/benchmarks/export_benchmark_config.py#L243)